src/org/company/LargeClass.java

```diff
@@ -0,0 +1,25 @@
+package org.company;
+
+/**
+ * @since Current.Project.Version
+ */
+public class LargeClass
+{
```

Comment:
braces aren't consistent. We need to pick a style, I'm in favor of leaving this repo and braces on a new line, but am okay either way.